benchmarks/python/core.py

```diff
@@ -237,7 +237,7 @@ def _get_kernel_time(
         elapsed_cuda_time = (
             sum(
                 [
-                    event.self_cuda_time_total
+                    event.self_device_time_total
```

Comment:
I think you'll need to guard this with `NVF_TORCH_VERSION_GREATER`. Otherwise we won't be able to build with stable torch release

Reply:
wait... this is python. wasn't paying attention to that.

But how did it work through CI? we did test against stable torch version right. Is benchmark not part of CI on older torch version?

Reply:
For torch stable build & test in CI, we run python tests but not python benchmarks.

Reply:
Should we add the check for torch version then?

Reply:
I think we should. Otherwise we won't be able to run benchmark against old torch version.

Reply:
`torch==2.3` is still using `self_cuda_time_total`. Will add a version guard for torch 2.4.